ca/certificate-authority.go

```diff
@@ -433,36 +553,27 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
 		ca.log.Audit(fmt.Sprintf("Uncaught error, aborting and rolling back issuance: pem=[%s] err=[%v]", certPEM, err))
 		tx.Rollback()
-		return emptyCert, err
+		return
 	}
 
-	// Store the cert with the certificate authority, if provided
-	_, err = ca.SA.AddCertificate(certDER, regID)
+	logEventResult = "successful"
```

Comment:
Should we wait until after the cert is finalized before declaring success?  That is, until after tx.Commit()